OpenProblemLibrary/ma117DB/set5/srw2_9_63.pg

```diff
@@ -20,35 +20,39 @@ DOCUMENT();        # This should be the first executable line in the problem.
 
 loadMacros(
   "PGstandard.pl",
-  "PGchoicemacros.pl",
+  "PGML.pl",
+  "MathObjects.pl",
   "PGcourse.pl"
 );
 
-TEXT(beginproblem());
 $showPartialCorrectAnswers = 1;
 
+Context("Numeric");
+
 $a = random(2,9,1);
+$ans3 = Formula("sqrt(x)-$a");
+
+Context("Interval");
+
+$ansInt = Compute("[-$a, infinity)");
+
+BEGIN_PGML
+The function [`f(x)=(x+[$a])^2`] is not one-to-one. Choose the largest possible
+domain containing the number 100 so that the function restricted to the domain is one-to-one.
+
+The largest possible domain is [_________________]{$ansInt} (use interval notation).
```

Comment:
line 43: replace "interval notation" with [@ helpLink("interval","interval notation") @]*
(including the star at the end!)  This will create a link that points to instructions for entering intervals into webwork.